### PR TITLE
Popover timing tweaks

### LINKED ISSUE #47694: Column info popover is getting in my way too often
### Describe the bug

When I hover column header the column info popover is shown immediately
When I click column header first the column info popover flickers and then the modal is opened
https://www.loom.com/share/b0fa10a94b454bd492ec22f43b6bd2b9
https://www.loom.com/share/6c193fbb2b4b4b44b3811a81325c8117

### To Reproduce

1. Open any table
2. Hover a column header, you'll see the popover right away
3. Click on a column header, you'll first see the popover flickering

### Expected behavior

Column info popover should be shown only when you keep the mouse over the column header for a while, like 500 ms?
It should not flicker on click

### Logs

_No response_

### Information about your Metabase installation

```JSON
9f706d5
```

### Severity

Annoying

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by romeovs**
Closes https://github.com/metabase/metabase/issues/47694

### Description

Tweaks the timing of the info popovers to make them less intrusive.

### How to verify

Describe the steps to verify that the changes are working as expected.

1. Open a question or table and select the table visualisation
2. Hover the header items and make sure the information popover is visible, but only after hovering for 1s
3. Move the cursor to other header items and make sure the popover shows immediately
4. Click the headers and make sure the popover does not flicker

### Demo

https://github.com/user-attachments/assets/c6d17681-f1e8-411c-8cb7-d450700742ee

### Checklist
No functionality changed, just some timing

### CODE REVIEW

**Review by mazameli — APPROVED** [review]
🎉

File: frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx [hunk 1]
```diff
@@ -5,32 +5,40 @@ import useSequencedContentCloseHandler from "metabase/hooks/use-sequenced-conten
 import type { HoverCardProps } from "metabase/ui";
 import { HoverCard, useDelayGroup } from "metabase/ui";
 
-export const POPOVER_DELAY: [number, number] = [250, 150];
-export const POPOVER_TRANSITION_DURATION = 150;
+const POPOVER_TRANSITION_DURATION = 150;
+
+// Initially, the user will have to hover for this long to open the popover
+const POPOVER_SLOW_OPEN_DELAY = 1000;
```
Comment (kamilmielnik) [inline frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx:11]: As a user this is way too long for me.
I'm not used to popovers being displayed only after 1s of hovering.
If I don't see one after ~500ms I'm just going to assume it's not there.

Can we change it to 500, pretty please ? :pray:

File: frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx [hunk 2]
```diff
@@ -57,8 +65,10 @@ export function Popover({
     <HoverCard
       position={position}
       disabled={disabled}
-      openDelay={group.shouldDelay ? delay[0] : 0}
-      closeDelay={group.shouldDelay ? delay[1] : POPOVER_CLOSE_DELAY}
+      openDelay={
+        group.shouldDelay ? POPOVER_SLOW_OPEN_DELAY : POPOVER_FAST_OPEN_DELAY
```
Comment (kamilmielnik) [inline frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx:69]: I don't think we want to make this change globally.
The bug report was specifically about column info popover in table viz headers.

We also don't want the long delay for column info popover in notebook editor ([image](https://github.com/user-attachments/assets/3353cd1b-9f0e-406f-a1c5-a1e0892a1868)).

Comment (romeovs) [reply frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx:69]: I personally don't think there's a need to keep separate timings for the other info icons, but I'll let @mazameli or @mngr speak to that. I think the same arguments could be made for the other info icons accross the site.

FWIW, they currently have all the same timeout: 250ms. This PR increases that to 500ms.

Comment (mngr) [reply frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx:69]: @AlessioLaiso @ranquild could you guys please test it
I also don't think we should change the global delay settings. So if 250 is enough, then let's apply global settings to the column popover. If it's not enough, let's have them separate

Comment (romeovs) [reply frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx]: Okay, I've limited the increased delay to only happen on the column header popovers.

**Review by ranquild — APPROVED** [review]

**Pull request merged**